tests/cli/test_suite.py

```diff
@@ -1107,3 +1141,197 @@ def test_suite_list_with_multiple_suites(caplog, empty_data_context):
     assert "c.warning" in output
 
     assert_no_logging_messages_or_tracebacks(caplog, result)
+
+
+@mock.patch(
+    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+)
```

Comment:
This little beauty allows us to test that the correct statistics are sent in CLI commands.